fairscale/optim/oss.py

```diff
@@ -327,6 +327,9 @@ def _broadcast_state_dict(self) -> None:
             self.local_state_dict(), non_blocking=True, device=torch.device("cpu")
         )
 
+        # Tensor cannot be really empty, even if its size is meaningless
+        dummy_sync_tensor = torch.tensor([1], device=self._device)
```

Comment:
Shouln't the device of the dummy tensor match with the device of the stateful list?
What happens in `broadcast_object_list` when these have different devices?